<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>PPI</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Publisher</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Name:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="name_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>PRIEmitter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Connection Type:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="connectionType_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex" >
            <number>1</number>
           </property>
           <item>
            <property name="text" >
             <string>TCP</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Multicast</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Multicast Address:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="address_" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>237.1.2.50</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Connections:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLabel" name="connections_" >
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Emitter</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Rate:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="frequency_" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="keyboardTracking" >
              <bool>false</bool>
             </property>
             <property name="suffix" >
              <string> Hz</string>
             </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>99999</number>
             </property>
             <property name="value" >
              <number>500</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetRate_" >
             <property name="text" >
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>5</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Step:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="step_" >
             <item>
              <property name="text" >
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>5</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>20</number>
           </property>
           <item>
            <widget class="QCheckBox" name="loopAtEnd_" >
             <property name="text" >
              <string>Loop at End</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="simAz_" >
             <property name="text" >
              <string>Simulated Azimuth</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QPushButton" name="startStop_" >
             <property name="text" >
              <string>Start</string>
             </property>
             <property name="shortcut" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rewind_" >
             <property name="text" >
              <string>Rewind</string>
             </property>
             <property name="shortcut" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>File Stats</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Min Range:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="rangeMin_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxLength" >
          <number>8</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>Num Gates:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLineEdit" name="pulseSize_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxLength" >
          <number>8</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Max Range:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="rangeMax_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Beam Width:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="beamWidth_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Min Value:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="valueMin_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Num Messages:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLineEdit" name="pulseCount_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Max Value:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="valueMax_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>Frequency:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3" >
        <widget class="QLineEdit" name="fileFrequency_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles_" >
     <property name="title" >
      <string>&amp;Recent Files</string>
     </property>
    </widget>
    <addaction name="actionFileOpen_" />
    <addaction name="menuRecentFiles_" />
   </widget>
   <widget class="QMenu" name="menuEmitter_" >
    <property name="title" >
     <string>&amp;Emitter</string>
    </property>
    <addaction name="actionEmitterStart_" />
    <addaction name="actionEmitterRewind_" />
   </widget>
   <addaction name="menuFile_" />
   <addaction name="menuEmitter_" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionFileOpen_" >
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEmitterStart_" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Start</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEmitterRewind_" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Reset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRecentFilesClear_" >
   <property name="text" >
    <string>Clear &amp;Menu</string>
   </property>
   <property name="iconText" >
    <string>Clear Menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
